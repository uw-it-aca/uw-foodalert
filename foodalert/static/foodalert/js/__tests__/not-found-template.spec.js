"use strict";
import Vue from 'vue';
import BootstrapVue from 'bootstrap-vue'
Vue.use(BootstrapVue);

import {mount} from '@vue/test-utils';
import NotFoundTemplate from '../components/not-found-template.vue';

describe('NotFoundTemplate (404 Page)', function () {
        test('renders correctly', function () {
            var wrapper = mount(NotFoundTemplate, {
            });
            expect(wrapper.element).toMatchSnapshot();
        });
})

