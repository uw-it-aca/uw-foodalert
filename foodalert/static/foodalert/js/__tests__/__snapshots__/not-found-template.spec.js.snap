// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundTemplate (404 Page) renders correctly 1`] = `
<div
  class="container"
  style="padding: 40px 15px;"
>
  <div
    align-self="center"
    class="row justify-content-center"
  >
    <img
      src="../../img/hungryhusky.png"
    />
  </div>
   
  <br />
   
  <div
    align-self="center"
    class="row justify-content-center"
  >
    <h1>
      404 Not Found
    </h1>
  </div>
   
  <br />
   
  <div
    align-self="center"
    class="row justify-content-center"
  >
    <h2>
      The Page You Requested Could Not Be Found
    </h2>
  </div>
   
  <br />
   
  <div
    align-self="center"
    class="row justify-content-center"
  >
    <a
      class=""
      href="/"
      target="_self"
    >
      <button
        class="btn btn-outline-primary"
        type="button"
      >
        
                  Go To Homepage
              
      </button>
    </a>
  </div>
</div>
`;
